### PR TITLE
[MultiKueue] Document  MPI Operator managedBy field

### LINKED ISSUE #3313: [MultiKueue] Document how to use mpi-operator with managedBy
**What would you like to be added**:
Document how to use managedBy provided by the mpi-operator.

**Why is this needed**:
To show users how to use the mpi-operator with MultiKueue.

**Completion requirements**:

This enhancement requires the following artifacts:

Relates: #3289 

- [ ] Docs update

The artifacts should be linked in subsequent comments.

### PR DESCRIPTION
**Pull request by mszadkow**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Document how to use managedBy provided by the mpi-operator.
#### Which issue(s) this PR fixes:

Fixes #3313 

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: site/content/en/docs/tasks/run/multikueue/mpijob.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+### ManagedBy
+
+The feature allows you to disable the MPI Operator and delegate reconciliation of that job to the Kueue controller.
+In order to change the controller that reconciles the job to the Kueue you need to set a value of that field to `kueue.x-k8s.io/multikueue`. 
+
+However the `spec.runPolicy.managedBy` field is defaulted to `kueue.x-k8s.io/multikueue` automatically if following conditions are met.
+
+The `kueue.x-k8s.io/queue-name` annotation of the mpijob job points to a Local Queue, whose corresponding Cluster Queue uses the Multi Kueue admission check.
```
Comment (mimowo) [inline site/content/en/docs/tasks/run/multikueue/mpijob.md:37]: Maybe we can rephrase a bit, for example say: 

{{% alert title="Note" color="primary" %}}
Note: that Kueue defaults the `spec.runPolicy.managedBy` field to `kueue.x-k8s.io/multikueue` on the 
management cluster for MPIJob. This allows the MPIJob controller to ignore the Jobs managed by MultiKueue on the management cluster, and in particular skip Pod creation. The pods are created and
the actual computation will happen on the mirror copy of the Job on the selected worker cluster. The mirror copy of the Job does not have the field set.
{{% /alert %}}

**Review by mimowo — COMMENTED** [review]
LGTM overall, proposed rephrase to the section on managedBy,

File: site/content/en/docs/tasks/run/multikueue/mpijob.md (hunk 1 above)
Comment (mszadkow) [reply site/content/en/docs/tasks/run/multikueue/mpijob.md:37]: Sounds good, but can we drop the `note` as this is basically the whole paragraph?

Comment (mimowo) [reply site/content/en/docs/tasks/run/multikueue/mpijob.md:37]: I would suggest to replace the whole paragraph with the note just

**Review by mimowo — COMMENTED** [review]
/lgtm
/approve

File: site/content/en/docs/tasks/run/multikueue/mpijob.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Run MPI Jobs in Multi-Cluster"
```
Comment (tenzen-y) [inline site/content/en/docs/tasks/run/multikueue/mpijob.md:2]:
```suggestion
title: "Run MPIJobs in Multi-Cluster"
```

This is exactly MPIJob.

File: site/content/en/docs/tasks/run/multikueue/mpijob.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+linkTitle: "MPIJob"
+weight: 3
+date: 2024-10-25
+description: >
+  Run a MultiKueue scheduled MPI Jobs.
```
Comment (tenzen-y) [inline site/content/en/docs/tasks/run/multikueue/mpijob.md:7]:
```suggestion
  Run a MultiKueue scheduled MPIJobs.
```

File: site/content/en/docs/tasks/run/multikueue/mpijob.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+{{% alert title="Note" color="primary" %}}
+Note: While both MPI Operator and Training Operator must be running on the same cluster, there are special steps that has to be applied to Training Operator deployment.
+See [Working alongside MPI Operator](/docs/tasks/run/multikueue/kubeflow#working-alongside-mpi-operator) for more details.
+{{% /alert %}}
+
+See [MPI Operator Installation](https://www.kubeflow.org/docs/components/training/user-guides/mpi/#installation) for installation and configuration details of MPI Operator.
+
+## MultiKueue integration
+
+Once the setup is complete you can test it by running a MPI Job [`sample-mpijob.yaml`](/docs/tasks/run/kubeflow/mpijobs/#sample-mpijob).
```
Comment (tenzen-y) [inline site/content/en/docs/tasks/run/multikueue/mpijob.md:27]:
```suggestion
Once the setup is complete you can test it by running a MPIJob [`sample-mpijob.yaml`](/docs/tasks/run/kubeflow/mpijobs/#sample-mpijob). 
```

**Review by tenzen-y — COMMENTED** [review]
@mszadkow Could you address these comments?

File: site/content/en/docs/tasks/run/multikueue/mpijob.md (hunk 4 above)
Comment (mszadkow) [reply site/content/en/docs/tasks/run/multikueue/mpijob.md:27]: OK, will handle that

**Pull request merged**